src/coreclr/vm/class.cpp

```diff
@@ -1134,6 +1134,22 @@ void ClassLoader::LoadExactParents(MethodTable* pMT)
         EnsureLoaded(TypeHandle(pMT->GetCanonicalMethodTable()), CLASS_LOAD_EXACTPARENTS);
     }
 
+    if (pMT->GetClass()->HasRVAStaticFields())
+    {
+        ApproxFieldDescIterator fdIterator(pMT, ApproxFieldDescIterator::STATIC_FIELDS);
+        FieldDesc* pFD = NULL;
+        while ((pFD = fdIterator.Next()) != NULL)
+        {
+            if (pFD->IsByValue() && pFD->IsRVA())
```

Comment:
Should we assert that if IsRVA, IsByValue? It seems like having an RVA field of a reftype should be invalid